Bottom: 893b324fecabee67ed00a8c0ef586e5a7997d7ca
Top:    4c22cb1fe8c648248fc097a062d4ef58f9252a2f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-24 21:27:13 -0400

Refresh of phytec

---

diff --git a/sound/soc/fsl/pcm030-audio-fabric.c b/sound/soc/fsl/pcm030-audio-fabric.c
index 0a50bbd..deb59e0 100644
--- a/sound/soc/fsl/pcm030-audio-fabric.c
+++ b/sound/soc/fsl/pcm030-audio-fabric.c
@@ -1,5 +1,6 @@
 /*
- * Phytec pcm030 driver for the PSC of the Freescale MPC52xx configured as AC97 interface
+ * Phytec pcm030 driver for the PSC of the Freescale MPC52xx
+ * configured as AC97 interface
  *
  * Copyright 2008 Jon Smirl, Digispeaker
  * Author: Jon Smirl <jonsmirl@gmail.com>
